pkgs/development/python-modules/crossplane/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "crossplane";
+  version = "0.5.7";
+  src = fetchFromGitHub {
+    owner = "nginxinc";
+    repo = "crossplane";
+    rev = "v${version}";
+    sha256 = "0lv3frfvnvz5wjxwh3mwy8nbypv4i62v4bvy5fv7vd6kmbxy1q9l";
+  };
+
```

Comment:
The upstream repo contains [tests](https://github.com/nginxinc/crossplane/tree/master/tests). 

Also, please add `pythonImportsCheck`.

Reply:
Done. Any way to get pytest find tests by itself?

Reply:
Normally pytest should find the tests directory by itself.

Can you use the pytestCheckHook?

Reply:
Thanks, that works.